test/data/compile-app-settings/purging-rules/project/targets.json

```diff
@@ -0,0 +1 @@
+{}
```

Comment:
Add the other purging config here so the test makes sure they merge together nicely.

Reply:
I added another test to test merging, since both merging and creating from scratch are equally likely to be broken IMO